README.md

```diff
@@ -866,6 +919,17 @@ The custom options are:
   When disabled, it will forward the user to the cross-device flow in order to attempt to capture a video in another device. If the user is already in a mobile device and it does not support
   MediaRecorder, the unsupported browser error will be shown.
 
+#### auth
+
+This is Authentication step. If you have followed the guidelines specific to including auth, you'll have this step made available. In here, a loading screen is presented to the user to fetch all necessary resources to perform authentication.
+
+After all resources are loaded, the session is initialised, and the authentication check begins. An oval frame of the camera will be present (if camera permissions are provided) and actionable elements will render, asking the user to place their face in frame, followed up by a different set of instructions for them to follow to successfully authenticate the user.
+
+If the user is not a match, or conditions are not good to successfuly authenticate, they will be asked to retry authentication. If authentication is not possible, the page will rollback to the previous step.
```

Comment:
What implies "authentication is not possible"? Is it a certain number of unsuccessful attempts?

Reply:
OK, I think we say that in one of the config setting descriptions but it may be worth reiterating it here (i.e. describe what it means to be not possible, what the default is, what the user can do to change that)